bug: panic: open defsec\policies: file does not exist (Azure DevOps on Windows 2019)

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

`tfsec` task fails in Azure DevOps pipelines



**To Reproduce**
Steps to reproduce the behavior:
1. run  `tfsec` scan in Azure DevOps on Windows 2019 agent (`windows-latest`)
2. See an error 
```
panic: open defsec\policies: file does not exist

goroutine 1 [running]:
github.com/aquasecurity/defsec/rego.init.0()
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.19.1/rego/embed.go:20 +0x414
##[error]Cmd.exe exited with code '2'.
```

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
`tfsec` runs

**System Info**
<!-- complete the following -->
 - tfsec version: `v1.9.1`
 - terraform version: `v1.1.7`
 - OS: `windows-2019` and `windows-2022`

**Additional context**
<!-- Add any other context about the problem here. -->
Azure DevOps task
```bash
tfsec terraform/ --format JUnit --out tfsec-report-$(Build.BuildId).xml
```

--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,7 @@ go 1.16
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.2
 	github.com/Masterminds/semver v1.5.0
-	github.com/aquasecurity/defsec v0.19.2
+	github.com/aquasecurity/defsec v0.19.3
 	github.com/hashicorp/go-version v1.4.0
 	github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
 	github.com/liamg/clinch v1.5.6

